charts/gateway/templates/node-properties-secret.yaml

```diff
@@ -0,0 +1,17 @@
+{{ if not .Values.disklessConfig.enabled }}
```

Comment:
Need to set apiVersion. Getting:
```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: apiVersion not set
```

Reply:
Addressed in https://github.com/CAAPIM/apim-charts/pull/318/commits/0d5189e9571af9724fe57b2d231e71104ec93812